sw/host/opentitantool/src/command/lc.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use clap::{Args, Subcommand};
+use humantime::parse_duration;
+use serde_annotate::Annotate;
+use std::any::Any;
```

Comment:
nit: commonly std (core,alloc) imports are placed on a separate group.

Reply:
fixed in follow on (#20288).